test/cli/cli_test.bzl

```diff
@@ -65,10 +65,6 @@ def update_test():
     ]
     native.test_suite(
         name = "update",
-        tags = [
-            "manual",
-            "external",
-            "local",
-        ],
```

Comment:
Given https://docs.bazel.build/versions/main/test-encyclopedia.html#tag-conventions I don't think we want to remove "external" (unless we *do* want caching for this test).  https://docs.bazel.build/versions/main/be/common-definitions.html#common.tags also suggests "local" has a useful meaning too...

Reply:
They do have a special meaning for a normal rule, but as far as I can tell the only tag that has a special meaning for a `test_suite` is `"manual"`. I verified this by re-running the target locally, and am pretty sure that the only thing that keeping these tags here would do is to add additional targets to this test suite that were tagged either `"external"` or `"local"`.